### ISSUE TITLE
Nginx & SFTPGO Services not running after 0.17.5 update

### ISSUE THREAD

**Issue by d-trax** [AzuraCast/AzuraCast#5878]
### Installation Method

Ansible Installation

### AzuraCast Release Channel

Stable Channel

### Current AzuraCast Version

0.17.5

### What happened?

Hi, just updated to 0.17.5 and now I have some errors regarding Nginx and SFTPGO services, which they show that they aren't running. When I go to Administration panel and see the services the Nginx keeps turning green-red-green-red and etc. The SFTPGO service doesn't start at all, but when I check on my ssh it says that it's active and running. Nginx works fine as the station's page loads everytime, but it keeps showing me that it's starting and stopping by itself. Checked the logs and didn't show any errors. On the Supervisord log there is this:

```
2022-11-23 20:15:48,016 INFO success: mariadb entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
2022-11-23 20:15:48,016 INFO success: nginx entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
2022-11-23 20:15:49,518 INFO exited: nginx (exit status 1; not expected)
2022-11-23 20:15:50,523 INFO spawned: 'nginx' with pid 4859
```
it happends every second.

**Comment by BusterNeece** [comment]
@d-trax Interesting to see just those two services failing...let's check the logs.

From `./docker.sh bash`, you can do `tail -f /var/azuracast/www_tmp/service_nginx.log` (or `service_sftpgo.log`) and we can investigate based on your findings there.

**Comment by d-trax** [comment]
Hi, 
Sorry for the late reply. there aren't any logs in /var/azuracast/www_tmp/ and the docker.sh bash command returns permission denied (run via sudo). 
Just installed Ubuntu 22.04 and did a clean Azuracast install (0.17.6) and it's the same, but at least it's only showing that SFTPGO service isn't working. 
When I click the restart button on SFTPGO this is the error i'm getting

**Comment by BusterNeece** [comment]
@d-trax Try going to the System Administration page, then "Log Viewer". You should see service logs, including one for SFTPGo. Check that log.

**Comment by BusterNeece** [comment]
@gunsar There's an sftpgo service log in the Log Viewer in System Administration...what does it say?

**Comment by CyberMixRo** [comment]
I have the same issue, impossible to install and use Ansible version, sftpgo fail to start.

**Comment by NicoxDJ** [comment]
@BusterNeece Any new fixes about this problem? Just updated from 0.17.3 to 0.17.7, and i have the same problem like the guys above. There is no tab with SFTPGo logs, and mariadb and some other processes keeps crashing and respawning
```
2023-04-02 01:36:57,155 INFO spawned: 'mariadb' with pid 2974
2023-04-02 01:36:58,211 INFO success: mariadb entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
2023-04-02 01:37:27,564 INFO exited: mariadb (exit status 1; not expected)
2023-04-02 01:37:27,820 INFO spawned: 'mariadb' with pid 3020
2023-04-02 01:37:27,849 INFO exited: station_1_nowplaying (exit status 0; expected)
2023-04-02 01:37:27,866 INFO spawned: 'station_1_nowplaying' with pid 3022
2023-04-02 01:37:29,156 INFO success: mariadb entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
2023-04-02 01:37:29,156 INFO exited: station_2_nowplaying (exit status 0; expected)
2023-04-02 01:37:30,161 INFO spawned: 'station_2_nowplaying' with pid 3027
2023-04-02 01:37:33,268 INFO exited: php-worker (exit status 0; expected)
2023-04-02 01:37:34,273 INFO spawned: 'php-worker' with pid 3034
2023-04-02 01:37:35,406 INFO success: php-worker entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
2023-04-02 01:37:38,115 INFO success: station_1_nowplaying entered RUNNING state, process has stayed up for > than 10 seconds (startsecs)
2023-04-02 01:37:40,382 INFO success: station_2_nowplaying entered RUNNING state, process has stayed up for > than 10 seconds (startsecs)
9 INFO success: php-worker entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
2023-04-02 01:32:50,588 INFO exited: mariadb (exit status 1; not expected)
2023-04-02 01:32:50,913 INFO spawned: 'mariadb' with pid 2696
2023-04-02 01:32:51,970 INFO success: mariadb entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
2023-04-02 01:33:21,327 INFO exited: mariadb (exit status 1; not expected)
2023-04-02 01:33:21,703 INFO spawned: 'mariadb' with pid 2742
2023-04-02 01:33:22,778 INFO success: mariadb entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
2023-04-02 01:33:52,120 INFO exited: mariadb (exit status 1; not expected)
2023-04-02 01:33:52,438 INFO spawned: 'mariadb' with pid 2756
```
